### PR TITLE
fix(web): bugfix/A2-808-remove-change-links-for-not-received-lpaq

### PR DESCRIPTION
**Pull request by kandcdev**
## Describe your changes
https://pins-ds.atlassian.net/browse/A2-808

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/appeals/appeal-details/__tests__/appeal-details.test.js [hunk 1]
```diff
@@ -1949,4 +1949,40 @@ describe('appeal-details', () => {
...
+		expect(unprettifiedElement.innerHTML).toContain(
+			`href="/appeals-service/appeal-details/3/inspector-access/change/lpa"`
+		);
+		expect(unprettifiedElement.innerHTML).toContain(
+			`href="/appeals-service/appeal-details/3/neighbouring-sites/change/affected"`
+		);
+		expect(unprettifiedElement.innerHTML).toContain(
+			`href="/appeals-service/appeal-details/3/safety-risks/change/lpa"`
+		);
+	});
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/__tests__/appeal-details.test.js:1987]: Ideally there should also be a unit test to check that the change links aren't rendered when the LPAQ hasn't been recieved

File: appeals/web/src/server/lib/mappers/appeal.mapper.js [hunk 2]
```diff
@@ -1564,3 +1582,11 @@ export function generateDecisionDocumentDownloadHtml(appealDetails, linkText = '
 
 	return html;
 }
+
+/**
+ * @param {string | undefined} lpaQStatus
+ * @returns {boolean}
+ */
+function checkWhetherToDisplayChangeLinksBasedOnLPAQStatus(lpaQStatus) {
```
Comment (james-hill-kc) [inline appeals/web/src/server/lib/mappers/appeal.mapper.js:1590]: Minor one - I'm generally in favour of long, descriptive symbol names, but this does feel a little bit verbose - would something like `shouldDisplayChangeLinksForLPAQStatus` be better?

**Review by james-hill-kc — COMMENTED** [review]
Looks great overall, just a couple of comments

File: appeals/web/src/server/appeals/appeal-details/__tests__/appeal-details.test.js (hunk 1 above)
Comment (kandcdev) [reply appeals/web/src/server/appeals/appeal-details/__tests__/appeal-details.test.js:1997]: done

File: appeals/web/src/server/lib/mappers/appeal.mapper.js (hunk 2 above)
Comment (kandcdev) [reply appeals/web/src/server/lib/mappers/appeal.mapper.js]: done

**Review by james-hill-kc — APPROVED** [review]

**Pull request merged**